integreat_cms/cms/templates/language_tree/language_tree_node_form.html

```diff
@@ -6,7 +6,7 @@
 {% load widget_tweaks %}
 
 {% block content %}
-<form method="post">
+<form method="post" id="content_form">
```

Comment:
The problem with using this id is that it also enables autosave, which we don't want for all the other forms.
So probably it would be better, to use a separate class or data attribute to determine the forms which should be used for the browser warning.